README.md

```diff
@@ -15,18 +15,17 @@ Follow these steps to add a new version:
 
 Generate API docs in Foreman:
 
-1. cd to foreman directory and checkout the relevant stable branch
-1. disable any plugins installed locally
-1. `APIPIE_RECORD=examples bundle exec rake test`
-1. `FOREMAN_APIPIE_LANGS=en bundle exec rake apipie:cache`
+1. Go to https://github.com/theforeman/foreman/actions/workflows/foreman.yml?query=branch%3AX.Y-stable
```

Comment:
3.16 pick of the action changes: https://github.com/theforeman/foreman/pull/10655